oauth2_http/javatests/com/google/auth/oauth2/ComputeEngineCredentialsTest.java

```diff
@@ -64,6 +68,38 @@
 
   private static final URI CALL_URI = URI.create("http://googleapis.com/testapi/v1/foo");
 
+  public static final String standardIdToken =
```

Comment:
Constants should be uppercased and underscored.

Reply:
It might be worthwhile to add a comment on what's in these tokens so other's don't need to parse them to see the format.